frontend/public/components/filter-toolbar.tsx

```diff
@@ -294,49 +294,51 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
             )}
           </ToolbarItem>
         )}
-        <ToolbarItem className="co-filter-search--full-width">
-          <ToolbarFilter
-            deleteChipGroup={() => updateLabelFilter([])}
-            chips={[...labelFilters]}
-            deleteChip={(filter, chip: string) =>
-              updateLabelFilter(_.difference(labelFilters, [chip]))
-            }
-            categoryName="Label"
-          >
+        {!hideToolbar && !hideLabelFilter && (
```

Comment:
Won't this hide the name filter when only `hideLabelFilter` is specified?

Reply:
@spadgett updated logic here.